adiv5-swd.rb

```diff
@@ -54,7 +54,7 @@ def read(port, addr, opt={})
     if !opt[:count]
       reply = @drv.transfer(req)
     else
-      req[:count] = readcount
```

Comment:
From looking at cmsis-dap.rb, it seems that `:count` is the "new" way of passing this information...